### PR TITLE
Add last updated time to masjids

### LINKED ISSUE #6: Show last updated timestamp in relative time format

### PR DESCRIPTION
**Pull request by fayez-nazzal**
Add "Last Updated" column to masjids using relative time format when JavaScript is enabled. Otherwise, the ISO date will be displayed.

## JS enabled
image

## JS Disabled
image

### CODE REVIEW

**Comment by hammady** [comment]
The timestamp is always the same for all prayers in the same masjid. We need to show it only once per masjid. Moreover, with js enabled we need it to be relative (e.g updated 2 hours ago, 2 day ago, Tuesday, last week...). 
With no Js let's make it clear that it is in UTC.

**Comment by fayez-nazzal** [comment]
> The timestamp is always the same for all prayers in the same masjid. We need to show it only once per masjid. Moreover, with js enabled we need it to be relative (e.g updated 2 hours ago, 2 day ago, Tuesday, last week...). With no Js let's make it clear that it is in UTC.

@hammady  Updated, please check

File: generator/svelte/src/lib/utils.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+dayjs.extend(relativeTime);
+
+export const convertToRelativeTime = (isoDate: string) => dayjs().from(isoDate);
```
Comment (hammady) [inline generator/svelte/src/lib/utils.ts]: I like the format shown [here](https://day.js.org/docs/en/display/from-now). Can we use `fromNow` instead?

Comment (fayez-nazzal) [reply generator/svelte/src/lib/utils.ts]: Sure

**Review by hammady — APPROVED** [review]

**Pull request merged**